cloudinit/config/cc_growpart.py

```diff
@@ -316,6 +342,22 @@ def resize_devices(resizer, devices):
                          "failed to resize: disk=%s, ptnum=%s: %s" %
                          (disk, ptnum, e),))
 
+        if is_lvm and type(resizer).__name__ == "ResizeGrowPart":
```

Comment:
Quick comment on this line: I'm pretty sure there's a better to do check if `resizer` is `ResizeGrowPart`.

Reply:
I think just:

```python
if isinstance(resizer, ResizeGrowPart):
```